pkg/identityserver/blacklist/builtin.go

```diff
@@ -15,148 +15,678 @@
+	"bots",
+	"broker",
+	"bug",
+	"business",
+	"cache",
+	"cadastro",
+	"calendar",
+	"call",
+	"campaign",
+	"cancel",
+	"captcha",
+	"career",
+	"careers",
 	"carreers",
+	"cart",
+	"categories",
+	"category",
 	"cdn",
+	"cgi",
+	"cgi-bin",
+	"changelog",
+	"chat",
+	"check",
+	"checking",
+	"checkout",
+	"cli",
 	"client",
+	"cliente",
 	"clients",
+	"code",
+	"codereview",
+	"collaborator",
+	"collaborators",
+	"comercial",
+	"comment",
+	"comments",
+	"communities",
 	"community",
+	"company",
 	"compare",
+	"compras",
 	"conference",
+	"config",
+	"configuration",
+	"connect",
+	"console",
 	"contact",
+	"contact-us",
+	"contact_us",
+	"contactus",
+	"contest",
+	"contribute",
+	"corp",
+	"create",
+	"css",
 	"customers",
+	"dashboard",
+	"data",
+	"db",
+	"default",
+	"delete",
+	"demo",
+	"design",
+	"designer",
+	"destroy",
 	"dev",
+	"devel",
+	"developer",
+	"developers",
+	"device",
+	"devices",
+	"diagram",
+	"diary",
+	"dict",
+	"dictionary",
+	"die",
+	"dir",
+	"direct_messages",
+	"directory",
+	"dist",
 	"dns",
 	"doc",
 	"docs",
+	"documentation",
+	"domain",
+	"download",
+	"downloads",
+	"ecommerce",
+	"edit",
+	"editor",
+	"editors",
+	"edu",
+	"education",
+	"email",
+	"employment",
+	"empty",
+	"end",
+	"end_device",
+	"end_devices",
+	"enterprise",
+	"entries",
+	"entry",
+	"error",
+	"errors",
+	"eval",
+	"event",
 	"events",
 	"example",
 	"examples",
+	"exit",
+	"explore",
 	"extension",
 	"extensions",
+	"facebook",
 	"faq",
+	"favorite",
+	"favorites",
+	"feature",
+	"features",
+	"feed",
 	"feedback",
+	"feeds",
+	"file",
+	"files",
+	"first",
+	"flash",
+	"fleet",
+	"fleets",
+	"flog",
+	"follow",
+	"followers",
+	"following",
+	"forgot",
+	"form",
 	"forum",
 	"forums",
+	"founder",
+	"free",
+	"friend",
+	"friends",
+	"ftp",
+	"ga",
+	"gadget",
+	"gadgets",
+	"game",
+	"games",
 	"gateway",
+	"gatewayagent",
 	"gateways",
+	"gatewayserver",
+	"get",
+	"ghost",
+	"gift",
+	"gifts",
+	"gist",
 	"git",
+	"github",
+	"graph",
+	"group",
+	"groups",
+	"gs",
 	"gtw",
 	"gtws",
+	"guest",
+	"guests",
+	"handler",
 	"help",
 	"home",
+	"homepage",
 	"homes",
+	"host",
+	"hosting",
+	"hostmaster",
+	"hostname",
+	"howto",
+	"hpg",
+	"html",
 	"http",
+	"httpd",
 	"https",
+	"i",
+	"iamges",
+	"icon",
+	"icons",
 	"id",
+	"idea",
+	"ideas",
 	"identity",
+	"identityserver",
+	"image",
+	"images",
+	"imap",
+	"img",
+	"index",
+	"indice",
+	"info",
+	"information",
+	"inquiry",
+	"instagram",
+	"intranet",
+	"invitations",
+	"invite",
+	"ipad",
+	"iphone",
+	"irc",
+	"is",
+	"issue",
+	"issues",
+	"it",
+	"item",
+	"items",
+	"java",
+	"javascript",
+	"job",
+	"jobs",
 	"join",
+	"joinserver",
+	"js",
+	"json",
+	"jump",
+	"knowledgebase",
+	"language",
+	"languages",
+	"last",
+	"ldap-status",
 	"learn",
+	"legal",
+	"license",
+	"link",
+	"links",
+	"linux",
+	"list",
+	"lists",
 	"local",
 	"localdomain",
 	"localhost",
+	"log",
+	"log-in",
+	"log-out",
+	"log_in",
+	"log_out",
+	"login",
+	"logout",
+	"logs",
+	"m",
```

Comment:
Let's also add `lorawan`, `lora`, `lorawan-stack`

Reply:
`m` is too short anyway